osgbuild/constants.py

```diff
@@ -9,7 +9,7 @@
 WD_QUILT = '_quilt'
 AFS_CACHE_PATH = '/p/vdt/public/html/upstream'
 AFS_CACHE_PREFIX = 'file://' + AFS_CACHE_PATH
-WEB_CACHE_PREFIX = 'http://vdt.cs.wisc.edu/upstream'
+WEB_CACHE_PREFIX = 'https://vdt.cs.wisc.edu/upstream'
```

Comment:
Just to note it here i get a 404 for this url

Reply:
Yeah that's fine -- looks like there's no index.html autogeneration. If you try a specific file like https://vdt.cs.wisc.edu/upstream/condor/8.9.9/condor-8.9.9-1.el7.src.rpm, it will work.